pkg/apis/serving/v1alpha2/kfservice_types.go

```diff
@@ -190,6 +190,8 @@ type EndpointStatusMap map[constants.KFServiceEndpoint]*StatusConfigurationSpec
 type KFServiceStatus struct {
 	duckv1beta1.Status `json:",inline"`
 	URL                string             `json:"url,omitempty"`
+	Traffic            int                `json:"traffic,omitempty"`
```

Comment:
Can we make this a string of the form "100-0", "90-10"? Would be much nicer from a kubectl describe perspective.

Reply:
Ehh nevermind, let's fix this from the kubectl side and not pollute our data model with the jsonpath bug ;)

Reply:
Yes that would be sad - better to have a clean data model.